plugins/jobs/src/js/pages/JobDetailPageContainer.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { componentFromStream, graphqlObservable } from "data-service";
+import { Observable, BehaviorSubject } from "rxjs";
+import gql from "graphql-tag";
+import { default as schema } from "#PLUGINS/jobs/src/js/data/JobModel";
+import Loader from "#SRC/js/components/Loader";
+import Page from "#SRC/js/components/Page";
+import RequestErrorMsg from "#SRC/js/components/RequestErrorMsg";
+import JobDetailPage from "./JobDetailPage";
+
+export const DIALOGS = {
+  EDIT: "edit",
+  DESTROY: "destroy"
+};
+
+const LoadingScreen = function() {
+  return (
+    <Page>
+      <Page.Header breadcrumbs={[]} />
+      <Loader />
+    </Page>
+  );
+};
+
+const ErrorScreen = function() {
+  return (
+    <Page>
+      <Page.Header breadcrumbs={[]} />
+      <RequestErrorMsg />
+    </Page>
+  );
+};
+
+const disabledDialog$ = new BehaviorSubject(null);
```

Comment:
I can't really check that work right, because the dialog is no longer on the menu. 

I think this is another reason to have at least all the itens in the menu working, even if they do nothing.

Reply:
You are right, these parts would be tangled in at a later PR, but I can remove it if you like.